internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -202,6 +201,7 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 		Variables:                      s.manifest.TaskConfig.Variables,
 		Secrets:                        convertSecrets(s.manifest.TaskConfig.Secrets),
 		Aliases:                        aliases,
+		HTTPSListener:                  s.httpsEnabled,
```

Comment:
How does this go-template data play with the CFN parameter `HTTPSEnabled`?

Reply:
CFN parameter `HTTPSEnabled` we don't use it in our template anymore, the only reason why we keep it is because our operational commands depend on this parameter.